apps/meteor/client/hooks/useRoomsList.ts

```diff
@@ -36,12 +41,12 @@ export const useRoomsList = (
 				sort: JSON.stringify({ name: 1 }),
 			});
 
-			const items = rooms.map((room: any) => {
-				room._updatedAt = new Date(room._updatedAt);
-				room.label = room.name;
-				room.value = room.name;
-				return room;
-			});
+			const items = rooms.map((room: any) => ({
+				...room,
```

Comment:
I don't think we need the whole room object here, since the `useXList` usually only needs the value, label and updated at values, this is something I've done wrong when I've created this function

Reply:
done